packages/airnode-examples/integrations/coingecko-template/config.example.json

```diff
@@ -54,7 +54,13 @@
     "http": [],
     "httpSignedData": []
   },
-  "templates": [],
+  "templates": [
+    {
+      "templateId": "0x02834eb43d56133982b7d6e5aa8b466c7ea4ba0fadf697698c1fee0996bba0fc",
+      "endpointId": "0xd9e8c9bcc8960df5f954c0817757d2f7f9601bd638ea2f94e890ae5481681153",
+      "encodedParameters": "0x3173000000000000000000000000000000000000000000000000000000000000636f696e49640000000000000000000000000000000000000000000000000000657468657265756d000000000000000000000000000000000000000000000000"
+    }
+  ],
```

Comment:
I'm a bit confused. I must have missed that, what is this `templates` field for? I presume that it's for defining templates outside the blockchain, right? So, why do we need `create-template-on-chain.ts` then?

Reply:
The template must exist on chain in order for you to make a template request. When you hardcode a template in config.json, Airnode will not fetch it from chain (so we avoid a blockchain call).